pkg/cluster/admin_actions.go

```diff
@@ -31,3 +34,10 @@ func (u *simpleUpgrader) ListVMHostnames(ctx context.Context, cs *api.OpenShiftM
 
 	return hostnames, nil
 }
+
+func (u *simpleUpgrader) RunCommand(ctx context.Context, cs *api.OpenShiftManagedCluster, scaleset, instanceID, command string) error {
+	return u.vmc.RunCommand(ctx, cs.Properties.AzProfile.ResourceGroup, scaleset, instanceID, compute.RunCommandInput{
+		CommandID: to.StringPtr("RunShellScript"),
```

Comment:
nit: `RunAdminShellScript`

Reply:
I don't think so - this is an Azure-defined constant: see `az vm run-command list --location eastus` for the available ones